### ISSUE TITLE
Number of tokens mismatch in prediction and bert_pred outputs

### ISSUE THREAD

**Issue by Sanjeev2487** [dmis-lab/biobert#87]
I want to fine-tune with more than one label in my training set. I did modify the output labels as following for my data. Training the evaluation went fine. But when I try to detokenize the length of the label of test and BERT predicted labels do not match. And now, I'm confused about what I would have missed changing.  
Labels in my traning set were as  as following:

 `["B-PROB", "B-TREA", "B-TEST", "I-PROB", "I-TREA", "I-TEST", "O", "X", "[CLS]", "[SEP]"]`

**Comment by AndreasSaka** [comment]
@Sanjeev2487 I had similar problems did you manage to change the detokenizer?

**Comment by Sanjeev2487** [comment]
I did the brute-force and saved the map of each token beforehand. And use at the inference time to get the labels back.

**Comment by AndreasSaka** [comment]
@Sanjeev2487 If you can share more or If you have time to explain that would be amazing!

**Comment by wonjininfo** [comment]
Hi everyone,
Sorry for the delay in reply. 
I fixed most of the token length related bugs and reflected them in the latest comment (801a50). Would you check it [here](https://github.com/dmis-lab/biobert/tree/801a50e36783b4e1b0517968a24e27ffdf72a01b)? 
However, you will still need to modify the code a bit for the multi-type NER task. 

Thanks!
